projects/batfish-common-protocol/src/main/java/org/batfish/common/Container.java

```diff
@@ -72,11 +70,14 @@ public boolean equals(Object o) {
       return false;
     }
     Container other = (Container) o;
-    return Objects.equals(_name, other._name) && Objects.equals(_testrigs, other._testrigs);
+    return Objects.equals(_name, other._name)
+        && Objects.equals(_createdAt, other._createdAt)
```

Comment:
two equal containers can be created at different times(also I think just comparing counts of testrigs and analyses is not very useful, we would better call APIs(if they exist) to get lists and compare them).